pkg/lvgl/contrib/lvgl.c

```diff
@@ -80,34 +85,74 @@ static void *_task_thread(void *arg)
 
 static void _disp_map(lv_disp_drv_t *drv, const lv_area_t *area, lv_color_t *color_p)
 {
-    if (!_dev) {
+    if (!_screen_dev->display) {
         return;
     }
 
-    disp_dev_map(_dev, area->x1, area->x2, area->y1, area->y2,
+    disp_dev_map(_screen_dev->display, area->x1, area->x2, area->y1, area->y2,
                  (const uint16_t *)color_p);
 
     LOG_DEBUG("[lvgl] flush display\n");
 
     lv_disp_flush_ready(drv);
 }
 
-void lvgl_init(disp_dev_t *dev)
+#ifdef MODULE_TOUCH_DEV
+static bool _touch_read(lv_indev_drv_t *indev_driver, lv_indev_data_t *data)
```

Comment:
```suggestion
/* based on https://github.com/lvgl/lvgl/tree/v6.1.2#add-littlevgl-to-your-project */
static bool _touch_read(lv_indev_drv_t *indev_driver, lv_indev_data_t *data)
```

That's valuable information, let's add that.

Feel free to squash again

Reply:
Added and directly squashed.